notebooks/Image_Analysis.ipynb

```diff
@@ -308,7 +361,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "horizontal_coeff, vertical_coeff, diagonal_coeff = get_wt_coeffs_var(img_dir_path)"
+    "horizontal_coeff, vertical_coeff, diagonal_coeff = get_wt_coeffs_var(img_dir_path)\n",
```

Comment:
Plot all coefficients by default. Also, add support for plotting more than one dataset.

Reply:
This is left on the user on how they choose to do the visualization. Provided code should be good enough to replicate for multiple datasets.

Reply:
One coefficient on its own doesn't provide the full picture I'm afraid.

Reply:
By replicating I meant this:
```
horizontal_coeff_a, _, _ = get_wt_coeffs_var(img_dir_path)
horizontal_coeff_b, _, _ = get_wt_coeffs_var(img_dir_path)
```